vault/core.go

```diff
@@ -2316,6 +2345,10 @@ func (c *Core) postUnseal(ctx context.Context, ctxCancelFunc context.CancelFunc,
 		return err
 	}
 
+	if err := c.handleVersionTimeStamps(ctx); err != nil {
```

Comment:
postUnseal is invoked on all nodes, but I think handleVersionTimeStamps was meant to be run only on active nodes.

Reply:
Moved this to the end of `(s standardUnsealStrategy).unseal`